.ng-dev/perf-tests/test-rerun.diff

```diff
@@ -0,0 +1,17 @@
+diff --git a/ng-dev/utils/test/g3.spec.ts b/ng-dev/utils/test/g3.spec.ts
+index a82c1b7a..8e0b24f8 100644
+--- a/ng-dev/utils/test/g3.spec.ts
++++ b/ng-dev/utils/test/g3.spec.ts
+@@ -29,9 +29,9 @@ describe('G3Stats', () => {
+   });
+ 
+   function setupFakeSyncConfig(config: GoogleSyncConfig): string {
+-    const configFileName = 'sync-test-conf.json';
+-    fs.writeFileSync(path.join(git.baseDir, configFileName), JSON.stringify(config));
+-    return configFileName;
++    const somethingelse = 'sync-test-conf.json';
++    fs.writeFileSync(path.join(git.baseDir, somethingelse), JSON.stringify(config));
++    return somethingelse;
```

Comment:
I know we've chatted a bit about the patching; and being able to "reverse" is great; but it's worth discussing further IMO. Maintaining these patches can probably quickly break?! so we'd need to pick files that rarely change.

The reversing of patches is a good point; but I believe `git reset` would also work right?

Reply:
But when we wire up more workflows in FW, or other repos, we should chat more I believe?

Reply:
Agreed. I will leave this as is for now within this repo, and we can see how we want to address the situation when we encounter it in the other repos.